angular.json

```diff
@@ -63,6 +63,7 @@
               "projects/demo/src/assets"
             ],
             "styles": [
+              "node_modules/font-awesome/css/font-awesome.min.css",
```

Comment:
Since we use scss, there's an easier way (IMO) to integrate it. See https://github.com/Ninja-Squad/globe42/blob/master/frontend/src/styles.scss